worker/reboot/reboot_test.go

```diff
@@ -67,7 +69,7 @@ func (s *rebootSuite) TestWorkerReboot(c *gc.C) {
 	w, err := reboot.NewReboot(client, names.NewMachineTag("666"), lock)
 	c.Assert(err, jc.ErrorIsNil)
 	err = workertest.CheckKilled(c, w)
-	c.Assert(err, gc.Equals, worker.ErrRebootMachine)
+	c.Assert(err, jc.Satisfies, func(e any) bool { return errors.Is(e.(error), worker.ErrRebootMachine) })
```

Comment:
I've just been using `c.Assert(errors.Is(err, worker.ErrRebootMachine), jc.IsTrue)`

Reply:
I got fed up as well, I'd rather have checker support: https://github.com/juju/testing/commit/d0b20a61ed5d9ef7f96064041061e7a1432332e3